contracts/dex/src/exchanges/exchange_resolver.rs

```diff
@@ -27,6 +29,8 @@ pub(crate) fn identify_exchange(value: &str) -> Result<&'static dyn Identify, De
         OSMOSIS => Ok(&Osmosis {
             local_proxy_addr: None,
         }),
+        #[cfg(feature = "kujira")]
```

Comment:
This doesn't need a feature flag, see how it's done here : https://github.com/AbstractSDK/adapters/blob/main/contracts/dex/src/exchanges/exchange_resolver.rs#L13

In order to allow for IBC interactions, the features should not prevent identification, but should prevent implementation if not on the right chain.

Reply:
done, ty for your feedback